docs/Gedis/README.md

```diff
@@ -1,50 +1,120 @@
+    ```
+
+    ```out
+    !jumpscale.example.wallet
+    ```
+    """
+    w = schema_out.new()
+
+    w.ipaddr = wallet['ipaddr']
+    w.addr = wallet['addr']
+    w.jwt = wallet['jwt']
+    return w
+```
+
+This actor method has 2 arguments, `wallet` and `schema_out`. With the docstring we ask gedis to validate that `wallet` is actually a valid instance of the schema named `jumpscale.example.wallet`. We also define the type of the return value with the `schema_out` argument and the docstring which specify that schema_out is also an instance of the `jumpscale.example.wallet`.
+
+When defining docstring like this, if the data received or sent back doesn't validate the schema, an error will be raised.
+
+## Automatic client generation
+
+Gedis doesn't requires to generate client stub or anything like that. During the connection, the client will receive the generated code for the actor from the server directly.
+This make the deployment and update of you client really easy since you don't have to give specific client code to your users. All the magic happens transparently when you connect to a server.
+
+## Quick start
+
+### Define server interface
+
+Creation of an actor at `/tmp/actor.py`:
+
+```python
+from Jumpscale import j
+
+JSBASE = j.application.JSBaseClass
+
+class actor(JSBASE):
+
+    def __init__(self):
+        JSBASE.__init__(self)
+
+    def ping(self):
+        return "pong"
+```
+
+Creation of the gedis service and load our actor:
+
 ```python
+# configure the server
+server = j.servers.gedis.configure(instance='test', port=8889, host='0.0.0.0', ssl=False, adminsecret='')
+# load a single actor
+server.actor_add('/tmp/actor.py', namespace='demo')
+# you can also load a directory that contains multiple actor files
+# let's imagine you have a directory structure like
+# tree test_actor/
+# test_actor/
+# ├── actor2.py
+# ├── actor.py
+# you can load all the actors with
+server.actors_add('/tmp/test_actor', namespace='demo')
 
-# Start gedis test server in tmux
-cmd = "js_shell 'j.servers.gedis.test_server_start()'"
-j.tools.tmux.execute(
-    cmd,
-    session='main',
-    window='gedis_test',
-    pane='main',
-    session_reset=False,
-    window_reset=True
-)
+# start the server
+server.start()
+```
 
-#then get a gedis client
-client = j.clients.gedis.get("test")
+### Get a python client
 
-# then you will see that the registered actors are loaded in the client 
-# so you can perform actions directly using the client 
-client.gedis_examples.echo("s")
+```python
+# create a client
+# during the connection, the client will receive the generated code for the actor
+client = j.clients.gedis.configure(instance='test', host='192.168.10.10', port=8889, namespace='demo', ssl=False)
 
+# use the client
+client.actor.ping()
+# result will be b'pong'
 ```
+
 **to see more usage examples please read the tests in [gedis_factory class](DigitalMeLib/servers/gedis/GedisFactory.py)**
 
+### Get a javaScript Client
```

Comment:
the javascript client is now moved for now to be static into openresty as the js client is just stubs for the actor methods